<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="PrintGraph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1051</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="NodeBtn">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>10</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить вершину</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EdgeBtn">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>90</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить ребро</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DelEdgeBtn">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>170</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить ребро</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DelNodeBtn">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>250</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить вершину</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TspBtn">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>330</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Коммивояжер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MatrixBtn">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>420</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вывод матрицы</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Задача коммивояжера</string>
    </property>
   </widget>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
